.cargo/config

```diff
@@ -0,0 +1,2 @@
+[alias]
+run-wasm = "run --release --package run-wasm --"
```

Comment:
```suggestion
run-wasm = "run --package run-wasm --"
```
The cost of compiling in release mode isn't worth the non-observable performance benefit.

Reply:
Another nit: the norm for multi-argument commands is to use an array of strings, with one string per argument, so you'd get `run-wasm = ["run", "--package", "run-wasm", "--"]`

Reply:
On my machine, just going visually from the time taken for the `Serving web on http://localhost:8000` message to appear on debug mode it takes about ~3s on release it takes ~100ms

That seems like a huge improvement to iterative builds and makes the up front --release build absolutely worth it.